### ISSUE TITLE
semantic-ui-less reset.overrides?

### ISSUE THREAD

**Issue by payneio** [justkey007/fomantic-ui-sass#6]
semantic-ui-less has a file in `/themes/default/globals/reset.overrides` which this project does not have. Was that intentional?

**Comment by justkey007** [comment]
Yes it is intentional because in the code write in LESS everything has been mixed and it is not easy to be able to locate the location of a code related to a certain behavior. I have refactored so.

**Comment by payneio** [comment]
Refactored? Where did you end up putting the reset styes then? Or did you just remove them?

**Comment by justkey007** [comment]
Reset styles ==> https://github.com/justkey007/fomantic-ui-sass/tree/master/src/definitions/globals
Variables ===> https://github.com/justkey007/fomantic-ui-sass/tree/master/src/themes/default/globals
![2019-12-09_231112](https://user-images.githubusercontent.com/36489637/70477569-6bfa1b80-1ad9-11ea-96c2-907d6a26db88.png)

**Comment by payneio** [comment]
Sorry... maybe I'm slow, but I don't see the normalize.css rules anywhere? https://github.com/Semantic-Org/Semantic-UI-LESS/blob/master/themes/default/globals/reset.overrides

**Comment by payneio** [comment]
The inclusion of `/*! normalize.css v7.0.0 | MIT License | github.com/necolas/normalize.css */` into Semantic-UI seems essential to the framework to make the theme behave consistently cross-platform.

**Comment by payneio** [comment]
Specifically, I see https://github.com/justkey007/fomantic-ui-sass/tree/master/src/definitions/globals, but I don't see https://github.com/Semantic-Org/Semantic-UI-LESS/blob/master/themes/default/globals/reset.overrides there.

**Comment by justkey007** [comment]
> The inclusion of `/*! normalize.css v7.0.0 | MIT License | github.com/necolas/normalize.css */` into Semantic-UI seems essential to the framework to make the theme behave consistently cross-platform.

Yes you are right I omitted it without knowing it. As I told you, a lot of the work has been done by automation. All the .overrides files have been deleted and replaced by .style.scss for the components. So I did not pay attention to the globals folder files. Help me integrate it if you have time. Thank you
